### ISSUE TITLE
Don't install a module named core

### ISSUE THREAD

**Issue by copy** [mirleft/ocaml-tls#346]
`tls` installs the following files:

```
.opam/4.04.0+flambda/lib/tls/core.annot
.opam/4.04.0+flambda/lib/tls/core.ml
.opam/4.04.0+flambda/lib/tls/core.cmt
```

These shadow Janestreet's Core library in ocp-browser and also cause [other problems](https://github.com/OCamlPro/ocp-index/issues/100).

**Comment by rgrinberg** [comment]
Is this still a problem? I don't see anything like that on the latest release of Tls:

```
$ ls $(ocamlfind query tls)
META
opam
opam.config
tls.a
tls.cma
tls.cmi
tls.cmt
tls.cmx
tls.cmxa
tls.cmxs
tls-lwt.a
tls-lwt.cma
tls_lwt.cmi
tls_lwt.cmti
tls_lwt.cmx
tls-lwt.cmxa
tls-lwt.cmxs
tls_lwt.mli
tls-mirage.a
tls-mirage.cma
tls_mirage.cmi
tls_mirage.cmti
tls_mirage.cmx
tls-mirage.cmxa
tls-mirage.cmxs
tls_mirage.mli
x509_lwt.cmi
x509_lwt.cmti
x509_lwt.cmx
x509_lwt.mli
```

**Comment by copy** [comment]
Indeed, seems to have been fixed some time between 0.7.1 and 0.8.
